CHANGELOG.md

```diff
@@ -27,7 +28,7 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - chaining of movenpc events did trigger fail events
 - effect event duration -1 did not result in infinite duration
 - translation in conversation got stored in default language
-- incorrect calculation of the width of strings when using characters with certain accents when doing pagination 
+- incorrect calculation of the width of strings when using characters with certain accents when doing pagination
```

Comment:
This need to be moved to the new changelog section for the next version

Reply:
Here is just an unnecessary whitespace in the previous changelog removed?

Reply:
It is in the 2.1.3 release changelog. It need to be in the latest